node/config.go

```diff
@@ -0,0 +1,17 @@
+package node
+
+import "github.com/celestiaorg/celestia-node/node/p2p"
+
+// Config is main configuration structure for a Node.
+// It combines configuration units for all Node subsystems.
+type Config struct {
+	P2P *p2p.Config
+}
+
+// EmptyConfig creates an empty Node Config.
+// Its main purpose is to fill out pointers not to run into nil pointer dereference.
+func EmptyConfig() *Config {
```

Comment:
can we make this private? Or instead, make a `DefaultConfig` function that returns a default config?

Reply:
Let's do DefaultConfig